mmseg/models/segmentors/seg_tta.py

```diff
@@ -39,11 +39,18 @@ def merge_preds(self, data_samples_list: List[SampleList]) -> SampleList:
                             ).to(logits).squeeze(1)
             else:
                 seg_pred = logits.argmax(dim=0)
-            data_sample = SegDataSample(
-                **{
-                    'pred_sem_seg': PixelData(data=seg_pred),
-                    'gt_sem_seg': data_samples[0].gt_sem_seg
-                })
```

Comment:
I'm afraid not. seg_pred is the final prediction, we will get the last prediction in data_samples if we delete it. But we can still simplify the code by
```
data_sample.set_data({'pred_sem_seg': PixelData(data=seg_pred)})
if hasattr(data_samples[0], 'gt_sem_seg'):
    data_sample.set_data({'gt_sem_seg': data_samples[0].gt_sem_seg})
```